R/splatPop-simulate.R

```diff
@@ -151,9 +170,14 @@ splatPopSimulateMeans <- function(vcf = mockVCF(),
     samples <- colnames(VariantAnnotation::geno(vcf)$GT)
     conditions <- splatPopDesignConditions(params, samples)
 
-    # Genes from key or gff or mock (in that order)
+    # Genes from key if provided, or from empirical data or simulated from gff 
```

Comment:
It might be nice to add a progress message saying where the means are coming from so it's clear to the user.